tests/test_py38_cp38_wheel.nix

```diff
@@ -1,11 +1,10 @@
 {
-  mach-nix ? import ../. {
-    python = "python38";
-  },
+  mach-nix ? import ../. {},
   ...
 }:
 with builtins;
 mach-nix.mkPython {
+  python = "python38";
```

Comment:
This test was failing when run via `all-tests.nix` (and now `make-tests.nix`), since a different mach-nix was being passed in.